README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
+
+## Getting started
+
+To preview or build the website, we need to use the terminal.
```

Comment:
It's the royal 'we' :)

Unless we change direction, isn't this a repo that will be forked and built upon? The 'we' is a bit clearer cut when the thing will be in the author's ownership. Collective code ownership and all that.